lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationClient.cpp

```diff
@@ -199,6 +199,20 @@ uint64_t GDBRemoteCommunicationClient::GetRemoteMaxPacketSize() {
   return m_max_packet_size;
 }
 
+bool GDBRemoteCommunicationClient::GetReverseContinueSupported() {
+  if (m_supports_reverse_continue == eLazyBoolCalculate) {
+    GetRemoteQSupported();
+  }
+  return m_supports_reverse_continue == eLazyBoolYes;
+}
+
+bool GDBRemoteCommunicationClient::GetReverseStepSupported() {
+  if (m_supports_reverse_step == eLazyBoolCalculate) {
+    GetRemoteQSupported();
+  }
```

Comment:
We don't use (aren't supposed to use?) braces on short statements like these: https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

```suggestion
  if (m_supports_reverse_continue == eLazyBoolCalculate)
    GetRemoteQSupported();
  return m_supports_reverse_continue == eLazyBoolYes;
}

bool GDBRemoteCommunicationClient::GetReverseStepSupported() {
  if (m_supports_reverse_step == eLazyBoolCalculate)
    GetRemoteQSupported();
```